src/Pie/index.tsx

```diff
@@ -62,19 +66,43 @@ const Pie: React.FC<PieProps> = ({
 
                 currentAngle = nextAngle + paddingAngle;
 
+                // Define el label
+                let labelText = '';
+                if (label === 'percent') {
+                    labelText = `${((value / totalValue) * 100).toFixed(1)}%`;
+                } else if (Array.isArray(label) && label[index]) {
+                    labelText = label[index];
+                } else {
+                    labelText = `${value}`;
+                }
+
+                // Aumentar el tamaño si el segmento está activo
```

Comment:
Probably remove comments, if you think they are needed then use English, please.

Reply:
✅